team-validator.js

```diff
@@ -215,6 +217,16 @@ class Validator {
 				return [`"${set.ability}" is an invalid ability.`];
 			}
 		}
+		nature = tools.getNature(set.nature);
+		if (nature.id && !nature.exists) {
+			if (tools.gen < 3) {
+				//gen 1-2 don't have natures, just remove them
```

Comment:
If you're going to warn instead of just removing invalid natures from future gens, it doesn't make sense not to do the same for past gens.